lang/abcl/Portfile

```diff
@@ -26,11 +26,15 @@ platforms           darwin
 master_sites        https://abcl.org/releases/${version}/
 distname            abcl-src-${version}
 
-checksums           md5     9bbdf32caa0ed33f89c065fe60d1a989 \
-                    sha1    c2851f214181a2d3f8a68bd88c18624fd101864f \
-                    rmd160  24076d5f6539dcf759114530966a6d3acfa2a252 \
-                    sha256  1d871ee2f6bcf991d5a6eff7ea5105ef808610db882604d4df0411e971ad257f \
-                    size    2613411
+livecheck.type      regex
+livecheck.url       ${homepage}
+livecheck.regex     abcl-src-(\[0-9.\]+)${extract.suffix}
+
+checksums           md5     50a5725e0ec2cf9836edf48df4a11b82 \
+                    sha1    728d6863a98f59d5a5feb329cff7be523d58b33e \
```

Comment:
no need for the `md5` and `sha1` checksums anymore - please remove.

Reply:
I have removed the checksums in question and untabified the file. If there's anything else to change, let me know.